### ISSUE TITLE
Two way data transmission

### ISSUE THREAD

**Issue by mamsoudi** [CongducPham/LowCostLoRaGw#135]
We're currently using your codes and they're working perfectly. We're able to send data from an end device to the the gateway, without any change to the codes or customizations.

As far as I understand, gateway is only receiving data and the end device is only sending data (if we ignore ACK). But can we make this two way? Can gateway also act as a data sender so that we can control end device?

What are the hardware requirements if there's any, or what changes should we be making if such thing is possible?

**Comment by Povolnov** [comment]
Yes, you can do it by using downlink features - https://github.com/CongducPham/LowCostLoRaGw/blob/master/gw_full_latest/README-downlink.md
